### PR TITLE
New logo and pictures

### PR DESCRIPTION
**Pull request by keithf4**
# Description  

Updated to new logo shortcode.

Added pgmonitor logo and architecture diagram.

Updated grafana gif

Added descriptions of grafana dashboards

## Type of change  
Please check all options that are relevant  
- [ ] Bug fix (change which fixes an issue)  
- [ ] Feature (change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update only  

# How Has This Been Tested?  
**Tested Configuration**:  
- [ ] Ansible, Specify version(s):  
- Installation method:  
    - [ ] Binary install from source, version:  
    - [ ] OS package repository, distro, and version:  
    - [ ] Local package server, version:  
    - [ ] Custom-built package, version:  
    - [ ] Other:  
- [ ] PostgreSQL, Specify version(s):  
- [x] docs tested with hugo version(s):  0.74.3

Tested with playbook(s):  

# Checklist:  
- [x] My changes generate no new lint warnings  
- I have made corresponding changes to:  
    - [ ] the documentation  
    - [ ] the release notes  
    - [ ] the upgrade doc

### CODE REVIEW

**Review by pgguru — DISMISSED** [review]

**Review by hunleyd — DISMISSED** [review]

File: hugo/content/grafana/_index.md [hunk 1]
```diff
@@ -106,3 +107,44 @@ The extras package takes care of putting all these files in place. If you did no
...
 provisioning = /etc/grafana/provisioning
 ```
+
+### Included Dashboards {#dashboards}
+
+pgMonitor comes with several dashboards ready to be used with automatic provisioning. They provide examples of using the metrics from the postgres_exporter and node_exporter. Since provisioned dashboards cannot be edited directly in the web interface, if any custom changes are desired, it is recommmended to make a copy of them and make your changes there.
+
+#### PostgreSQL Dashboards
+
+ * *Bloat Details* - Provides details on database bloat (wasted space). Provides overview and top-n statistics.
```
Comment (sharmay) [inline hugo/content/grafana/_index.md:117]: also show dashboard filename ?
A table might look better.

**Review by sharmay — CHANGES_REQUESTED** [review]

File: hugo/content/grafana/_index.md [hunk 2]
```diff
@@ -106,3 +107,44 @@ The extras package takes care of putting all these files in place. If you did no
 [paths]
 provisioning = /etc/grafana/provisioning
 ```
+
+### Included Dashboards {#dashboards}
```
Comment (sharmay) [inline hugo/content/grafana/_index.md:111]: Missing alerts dashboard

File: hugo/content/grafana/_index.md (hunk 1 above)
Comment (keithf4) [reply hugo/content/grafana/_index.md:117]: I'd had the filename before, but it looked weird in this format. Will try with a table.

Comment (keithf4) [reply hugo/content/grafana/_index.md]: Turned into table and added filenames. The filenames are almost exactly the same as the dashboard titles, so does look a little redundant. Can leave it in of consensus likes it tho.

File: hugo/content/grafana/_index.md [hunk 3]
```diff
@@ -106,3 +107,44 @@ The extras package takes care of putting all these files in place. If you did no
...
+
+ * *Network Details* - Provides details on network usage (utilization, traffic in/out, netstat, etc)
+
+ * *Overview* - The top level overview dashboard that provides links to the OS Overview, PostgreSQL Overview, ETCD, and Prometheus Alerts dashboards
+
+ * *OS Details* - Provides details on operating system metrics (cpu, memory, swap, disk usage). Links to Filesystem Details dashboard.
+
+ * *OS Overview* - Provides an overview that shows the up status of each system monitored by pgMonitor
+
+ * *Prometheus Alerts* - Provides a summary list of current and recent alerts that have fired in Prometheus
```
[the remark this answers is not in the document]
Comment (keithf4) [reply hugo/content/grafana/_index.md]: Removed the section titles and just put a small separator between types of dashboards. Put this one down in a sort of "general" section with etcd

File: hugo/content/grafana/_index.md (hunk 1 above)
Comment (sharmay) [reply hugo/content/grafana/_index.md]: Looks nice with table, my 2 cents filename are good to have.
Other ideas
- move this table to the top just after intro section
- Link each dashboard to a scaren shot

File: hugo/content/grafana/_index.md [hunk 4]
```diff
@@ -11,6 +11,7 @@ There are RPM packages available to [Crunchy Data](https://www.crunchydata.com)
 - [Upgrading](#upgrading)
 - [Setup](#setup)
     - [Linux](#setup-on-linux)
+- [Included Dashboards](#dashboards)
 
 ## Installation {#installation}
```
Comment (sharmay) [inline hugo/content/grafana/_index.md:16]: Move alertmanager section out of prometheus ?
http://localhost:1313/#installation
1. exporter
2. Prometheus
3. Alertmanager
4. Grafana
My 2 cents - with alertmanager HA it like a product

Comment (keithf4) [reply hugo/content/grafana/_index.md:40]: Made an issue to see if we can expand alertmanager docs and separate them out to their own 

https://github.com/CrunchyData/pgmonitor/issues/271

File: hugo/content/grafana/_index.md (hunk 1 above)
Comment (keithf4) [reply hugo/content/grafana/_index.md]: Moved the table to the top. 

Can see about adding more pictures in the future if still desired. Does mean having to keep up with design changes and appearance does change slightly with new major grafana versions.

**Review by hunleyd — APPROVED** [review]

**Review by pgguru — APPROVED** [review]

**Review by sharmay — APPROVED** [review]

**Pull request merged**